Ryujinx.Ui.Common/Configuration/ConfigurationFileFormat.cs

```diff
@@ -251,6 +251,11 @@ public class ConfigurationFileFormat
         /// </summary>
         public ShownFileTypes ShownFileTypes { get; set; }
 
+        /// <summary>
+        /// A list of file types to be hidden in the games List
```

Comment:
wrong comment copy pasta.
```suggestion
        /// Main window start-up position, size and state
```

Reply:
Fixed.